diff -Naur rdma-core-20/ibacm/prov/acmp/src/acmp.c spack-src/ibacm/prov/acmp/src/acmp.c
--- rdma-core-20/ibacm/prov/acmp/src/acmp.c	2018-09-08 19:08:47.000000000 +0000
+++ spack-src/ibacm/prov/acmp/src/acmp.c	2020-09-28 11:54:49.407172835 +0000
@@ -265,8 +265,8 @@
 /*
  * Service options - may be set through ibacm_opts.cfg file.
  */
-static char route_data_file[128] = ACM_CONF_DIR "/ibacm_route.data";
-static char addr_data_file[128] = ACM_CONF_DIR "/ibacm_hosts.data";
+static char route_data_file[256] = ACM_CONF_DIR "/ibacm_route.data";
+static char addr_data_file[256] = ACM_CONF_DIR "/ibacm_hosts.data";
 static enum acmp_addr_prot addr_prot = ACMP_ADDR_PROT_ACM;
 static int addr_timeout = 1440;
 static enum acmp_route_prot route_prot = ACMP_ROUTE_PROT_SA;
